shared/chat/inbox/row/big-team-channel/index.js

```diff
@@ -100,15 +115,7 @@ const styles = styleSheetCreate({
     isElectron: desktopStyles.clickable,
   }),
   selectedChannelBackground: {
-    ...globalStyles.flexBoxRow,
-    ...(isMobile ? globalStyles.fillAbsolute : {width: '100%'}),
```

Comment:
i don't think this is safe to get rid of. you'll no longer have correct wrapping

Reply:
We need wrapping on the channel in the inbox?

Reply:
make a long channel name and make sure